src/index.html

```diff
@@ -22,6 +22,294 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__tabs">
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="navigation__tab">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <h2 class="card__label">
+          APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__rating-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-name">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <div class="card__button">
```

Comment:
Add the blank line between the blocks
```suggestion
      >
        <h2 class="card__label">
          APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
        </h2>
        
        <p class="card__code">Product code: 195434</p>
        
        <div class="card__rating">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          <p class="card__rating-reviews">Reviews: 5</p>
        </div>
        
        <div class="card__price">
          <p class="card__price-name">Price:</p>
          <p class="card__price-number">$2,199</p>
        </div>
        
```